libpod/storage.go

```diff
@@ -80,7 +80,7 @@ func (r *storageService) CreateContainerStorage(ctx context.Context, systemConte
 		if err != nil {
 			return ContainerInfo{}, err
 		}
-		img, err := istorage.Transport.GetStoreImage(r.store, ref)
+		_, img, err := istorage.ResolveReference(ref)
```

Comment:
This function looks strange.  Especially the `imageName != ""` condition above.

Reply:
tracked in https://github.com/containers/podman/issues/20411